src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -111,8 +111,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('hub_name', hub_name_type, options_list=['--name', '-n'], id_part='name')
         c.argument('etag', options_list=['--etag', '-e'], help='Entity Tag (etag) of the object.')
         c.argument('sku', arg_type=get_enum_type(IotHubSku),
-                   help='Pricing tier for Azure IoT Hub. Default value is F1, which is free. '
-                        'Note that only one free IoT hub instance is allowed in each '
+                   help='Pricing tier for Azure IoT Hub. '
```

Comment:
Can you also show up the default value in help message?
Currently you add default value for sku in `iot_hub_create` function. To add the default information in help, I will recommend to add `default=IotHubSku.s1.value` in argument.

Reply:
The default value is dynamically determined already as part of Knack/cli-core from setting the arg value in the bound function.  Before this change, the default value was in the argument description twice. @zhoxing-ms made the same comment earlier, which was resolved with a screenshot from @anusapan showing the default description is there.

Reply:
Knack automatically shows the default value according to https://github.com/Azure/azure-cli/blob/511e5d638908abd1fe29bf23260748349322b783/src/azure-cli/azure/cli/command_modules/iot/custom.py#L368-L369